o3po/includes/class-o3po-publication-type.php

```diff
@@ -549,6 +548,12 @@ protected function save_meta_data( $post_id ) {
 
     }
 
+    private function get_doi_suffix( $doi_suffix_template, $doi_suffix, $date_published, $volume, $pages ){
```

Comment:
The `get_XXX()` methods in that class usually return information about individual posts. Can you find a better name for this function and add docstring block, please?

Reply:
Sure. Good to know that the changes can go back upstream!

Reply:
This request is now obsolete after the refactor